fuse_examples/imaging/classification/mnist/simple_mnist_starter.py

```diff
@@ -166,9 +165,8 @@ def run_train(paths: dict, train_params: dict):
         default_root_dir=paths["model_dir"],
         max_epochs=train_params["trainer.num_epochs"],
         accelerator=train_params["trainer.accelerator"],
-        strategy=train_params["trainer.strategy"],
+        strategy="auto",
```

Comment:
setting auto doesn't work for <2.0.0 versions.
Let's just not set it for now? (should work for both versions)

Reply:
Done :)